index.html

```diff
@@ -31,12 +31,145 @@ <h1 class="about-title">I’m Martin.Glad to see you!</h1>
         </p>
         <p class="connection">LET’S CONNECT</p>
         <ul>
-            <li><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
+            <li ><a href="#"><img src="image/Vector (15).png" alt="twitter"></a></li>
             <li><a href="#"><img src="image/Linkedin icon.png" alt="linkedin"></a></li>
             <li><a href="#"><img src="image/Vector (11).png" alt="mail"></a></li>
             <li><a href="#"><img src="image/Vector (12).png" alt="github"></a></li>
             <li><a href="#"><img src="image/Vector (13).png" alt="peace"></a></li>
         </ul>
     </section>
+    <!-- WORK SECTION -->
+    <section class="work">
+        <!-- CARD 1 -->
+        <article class="card-1">
+                <img src="image/Snapshoot Portfolio.png" alt="image1" >
+                <h3 class="tonic">Tonic</h3>
+                <div class="canop">
+                    <p class="can-info">CANOPY</p>
+                    <div class="point"></div>
+                    <p class="back">Back End Dev</p>
+                    <div class="point"></div>
+                    <p class="back">2015</p>
+                </div>
+                <p class="paranic">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul class="languages">
+                    <li><a href="#">HTML</a></li>
+                    <li><a href="#">CSS</a></li>
+                    <li><a href="#">JavaScript</a></li>
+                </ul>
+                <button class="see">See project</button>
+        </article>
```

Comment:
As requested by the previous reviewer, For each project

* Kindly align the content of the project cards with the `flexbox` 
* i.e. we use `display: flex` on the container `article` in order to layout its content. 🤓💪

 This will give you more control over the alignment and spacing between items.